engine/src/widget.cpp

```diff
@@ -389,7 +415,8 @@ bool MCWidget::getprop(MCExecContext& ctxt, uint32_t p_part_id, Properties p_whi
 		case P_LAYER:
 		case P_SCRIPT:
 		case P_PARENT_SCRIPT:
-		case P_NUMBER:
+        case P_SCRIPT_STATUS:
```

Comment:
This patch exposed that widget scriptStatus was being treated as a custom property

Reply:
yes, particularly with this PR in limbo ;-)

Reply:
https://quality.livecode.com/show_bug.cgi?id=21545